src/repr/Cargo.toml

```diff
@@ -34,6 +34,7 @@ mz-proto = { path = "../proto" }
 num-traits = "0.2.15"
 num_enum = "0.5.7"
 ordered-float = { version = "3.4.0", features = ["serde"] }
+paste = "1.0.9"
```

Comment:
I think this can go into `dev-dependencies` because it's only used in a test?

Reply:
Yes, ty!